src/main/java/de/hpi/bpt/argos/common/Observable.java

```diff
@@ -0,0 +1,20 @@
+package de.hpi.bpt.argos.common;
+
+/**
+ * This interface represents observable objects.
+ * @param <Observer> - the class of the observers
+ */
+public interface Observable<Observer> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00119)